doc/releases/changelog-dev.md

```diff
@@ -110,7 +110,7 @@
   ...     (x, Y), history = opt.step(qnode, x, Y, spectra=spectra, full_output=True)
   ...     print(f"New cost: {np.round(qnode(x, Y), 3)} reached via substeps {np.round(history, 3)}")
   New cost: 0.0 reached via substeps [-0.  0.  0.]
-  New cost: -1.0 reached via substeps [-0.276 -0.276 -1.   ]
+  New cost: -1.0 reached via substeps [-1. -1. -1.]
```

Comment:
How come this changed?

Reply:
I run the whole example and had different values only for this array.

Reply:
I see, but why is that?

Reply:
It was probably a copy paste typo